contrib/ansible/group_vars/ceph/ceph.yaml

```diff
@@ -1,22 +1,29 @@
-# Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-configFile: /etc/ceph/ceph.conf
-pool:
-  "rbd": # change pool name same to ceph pool, but don't change it if you choose lvm backend
-    diskType: SSD
-    AZ: default
-    accessProtocol: rbd
-    thinProvisioned: true
-    compressed: false
+# Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+configFile: /etc/ceph/ceph.conf
+pool:
+  rbd: # change pool name same to ceph pool, but don't change it if you choose lvm backend
+    storageType: block

```

Comment:
Is there any validation for the storageType?  What are the supported storageType?  block, file, object?

Reply:
It's discusssed a long time ago, and I just keep it to be compatible, but for its meaning IMO it's about block pool, file pool and object pool such stuffs. For detailed information, @wisererik Erik can solve your question.